cpp/src/groupby/hash/single_pass_functors.cuh

```diff
@@ -23,6 +23,91 @@
 #include <cuco/static_set_ref.cuh>
 
 namespace cudf::groupby::detail::hash {
+// TODO: TO BE REMOVED issue tracked via #17171
```

Comment:
Is this still relevant?

Reply:
Yes, #17171 should be addressed to avoid code duplication.